internal/utils/cli/error.go

```diff
@@ -0,0 +1,138 @@
+package cli
```

Comment:
idiomatic go strongly discourages util packages - and this is specific to errors not the cli.

what're your thoughts of making this its own package outside of internal maybe? I'd prefer not errors, but something along those lines.

Reply:
haha `errors` was a close second throughout all this...and it'd kill my ocd to have this bugfix be the sole thing that creeped outside of `internal/` 😬 

as a compromise, two other thoughts:

existing `terminal` package: this is basically the lowest-level package of imports, and can be used freely through the codebase.  my hesitation here is that it will ultimately mean the `cloud` packages (`atlas` and `realm`) have an import to the `terminal`/"UI"package which feels like an unnecessary coupling

new `cli/feedback` package: as a compromise (similar to what we've already done with the `cli/user` package so that it can be imported anywhere

refactor `cli/command_factory` to some nested `cli/runner` package, and keep all of these low-level concepts in the root `cli` package.  I like this most, and actually started down this path at one point during the reviews, but it is probably the biggest life.

any thoughts from your end hearing all of this?

Reply:
haha `errors` was a close second throughout all this...and it'd kill my ocd to have this bugfix be the sole thing that creeped outside of `internal/` 😬 

as a compromise, two other thoughts:

existing `terminal` package: this is basically the lowest-level package of imports, and can be used freely through the codebase.  my hesitation here is that it will ultimately mean the `cloud` packages (`atlas` and `realm`) have an import to the `terminal`/"UI"package which feels like an unnecessary coupling

new `cli/feedback` package: as a compromise (similar to what we've already done with the `cli/user` package so that it can be imported anywhere

refactor `cli/command_factory` to some nested `cli/runner` package, and keep all of these low-level concepts in the root `cli` package.  I like this most, and actually started down this path at one point during the reviews, but it is probably the biggest life.

any thoughts from your end hearing all of this?

Reply:
Since this is opt in, I think cli/feedback makes the most sense to me